frontendservice/src/views/ShopOwnerHome.vue

```diff
@@ -1,79 +1,171 @@
     </div>
-    <br />
-    <a class="button is-danger" v-if="isLoggedIn" @click="logout">Logout</a>
   </div>
 </template>
 
+<script src="https://cdn.jsdelivr.net/npm/sweetalert2@9"></script>
 <script>
+import shopOwnerNavbar from "../components/NavBar-ShopOwner";
+import axios from "axios";
+import Swal from "sweetalert2";
 import firebase from "firebase";
-
 export default {
-  name: "shopOwnerHome",
-
+  name: "MyShops",
+  components: {
+    shopOwnerNavbar
+  },
   data() {
     return {
-      name: "",
-      uid: "",
+      shopList: [],
+      searchResult: "",
+      renderComponent: true,
       isLoggedIn: false,
-      isVerified: false,
+      isVerified: false
     };
   },
+  methods: {
+    deleteshop: function(id) {
+      Swal.fire({
+        title: "Are you sure?",
+        text: "You won't be able to revert this!",
+        icon: "warning",
+        showCancelButton: true,
+        confirmButtonColor: "#3085d6",
+        cancelButtonColor: "#d33",
+        confirmButtonText: "Yes, delete it!"
+      }).then(result => {
+        if (result.isConfirmed == true) {
+          axios
+            .put("http://penguin.termina.linux.test:8080/shop/", {
+              shopId: id,
+              shopStatus: "DELETED"
+            })
+            .then();
+          this.renderComponent = false;
+          var self = this;
+          this.$nextTick(() => {
+            axios
+              .get(
+                "http://penguin.termina.linux.test:8080/shop/" +
+                  firebase.auth().currentUser.uid
+              )
+              .then(response => (self.shopList = response.data.shops));
+            self.renderComponent = true;
+          });
+        }
+      });
+    }
+  },
   created() {
     var user = firebase.auth().currentUser;
     if (user) {
-      this.name = user.displayName;
-      this.uid = user.uid;
       this.isLoggedIn = true;
       if (user.emailVerified) {
         this.isVerified = true;
       }
     }
+    var self = this;
+    setInterval(function() {
+      axios
+        .get(
+          "http://penguin.termina.linux.test:8080/shop/" +
+            firebase.auth().currentUser.uid
+        )
+        .then(response => (self.shopList = response.data.shops));
+    }, 2000);
   },
-  methods: {
-    logout: function() {
-      firebase
-        .auth()
-        .signOut()
-        .then(() => {
-          this.$router.replace("login");
-        });
-    },
-  },
+  computed: {
+    filteredShops: function() {
+      return this.shopList.filter(shop => {
+        var result = this.searchResult.toLowerCase();
+        var shopName = shop["shop"].shopName.toLowerCase();
+        return shopName.match(result);
+      });
+    }
+  }
 };
 </script>
 
 <style scoped>
-button {
-  margin-top: 1%;
-  width: 10%;
+* {
+  text-align: center;
+}
+button{
+  margin: 2%;
+}
+.control {
+  text-align: center;
+}
+#heading {
+  font-size: 2rem;
+}
+#search {
+  width: 50%;
```

Comment:
Nit. They need to be alphabetically arranged.

Reply:
done!